### ISSUE TITLE
complete list of hyperparameters used for FixMatch and MOCO

### ISSUE THREAD

**Issue by hzhz2020** [cvl-umass/ssl-evaluation#5]
Hi

Thanks for your interesting work! 
What is your final hyperparameters used for FixMatch and MOCO? it is not clear from the paper, for example did you use Adam as the default from FixMatch's original repo? what value did you use for weight decay? is there unlabeled loss warmup etc?

many thanks!
